bond/commands/select.py

```diff
@@ -1,11 +1,18 @@
 from .base_command import BaseCommand
+from .token import update_token
 import bond.database
 from ..cli.console import LogLine
+from bond.proto import get_async
```

Comment:
The imports could use better sorting. Not a big fan of from "..cli", but not sure how fix the path to avoid it.